.ci.yaml

```diff
@@ -1249,7 +1249,7 @@ targets:
           {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"}
         ]
       shard: web_tool_tests
-      subshard: web
+      subshard: "1_1"
```

Comment:
Target name `Linux web_tool_tests` => `Linux web_tool_tests_1_2`.
Same for the `Mac` one.
Seems the `Windows` one has been updated.

Reply:
`Linux web_tool_tests_1_1` right?

Reply:
Oh, this only has one shard. Then it should be good.

Reply:
question: should we just leave this field empty then?

Reply:
From a practical perspective it's fine to remove it
```
if (subshardName == null) {
  print('$kSubshardKey environment variable is missing, skipping sharding');
  return tests;
}
```

Reply:
Either LGTM. I don't have strong preference on it.

Reply:
But then again if in the future we want to further subshard linux/mac I think it will help to remember that this are already "subshard" ready?